examples/provisioner/windows.yaml

```diff
@@ -0,0 +1,57 @@
+# This example provisioner will provision instances running Windows Server OS
+
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: windows2019
+spec:
+  requirements:
+    - key: kubernetes.io/os
+      operator: In
+      values: ["windows"]
+    - key: node.kubernetes.io/windows-build
+      operator: In
+      values: ["10.0.17763"]
+  providerRef:
+    name: windows2019
+---
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: windows2019
+spec:
+  subnetSelector:
+    karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  securityGroupSelector:
+    karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  amiFamily: Windows2019
+  metadataOptions:
```

Comment:
Can you add -
```
    httpTokens: required
```
It is a good practise to enable IMDSv2.
Applicable to 2022 as well.